ntbs-service/Models/Entities/NotificationDisplay.cs

```diff
@@ -25,6 +25,12 @@ public partial class Notification
 
         public bool IsMBovis => DrugResistanceHelper.IsMbovis(DrugResistanceProfile);
 
+        public bool IsMBovisQuestionnaireComplete => DrugResistanceHelper.IsMBovisQuestionnaireComplete(
+            MBovisDetails.HasExposureToKnownCases,
+            MBovisDetails.HasUnpasteurisedMilkConsumption,
+            MBovisDetails.HasOccupationExposure,
+            MBovisDetails.HasAnimalExposure);
```

Comment:
Ah, I've just noticed that you're not fetching the whole `MBovisDetails` object in the job - is there a reason for that. Since you're fetching all the properties it won't make much of a difference from a performance perspective.

Reply:
Anyway this is only a very minor thing, so feel free to leave it as-is.

Reply:
So I've not done any performance testing but the idea here was to dodge getting the lists of exposure events of each type, just the booleans from the `MBovisDetails` page. I'm not sure if this implementation actually achieves this or not, and if it doesn't it would definitely be easier to just have an `MBovisDetails` in this projection object. Will this be automatically doing those joins and getting those list of events? And should we be worried about them?

Reply:
Right, I see what you were worried about. In this case, neither implementation ought to be fetching the lists of exposure events. In general, EF won't fetch navigation objects unless either (a) you tell it to explicitly using `Include`, (b) one object is configured to be an 'owner' of another, or (c) you configure eager loading in some other way.

Here, none of those conditions are met so you should be safe either way.

Reply:
Thanks for explaining that! I've changed to just using an `MBovisDetails` object here then